### PR TITLE
🐛 Fix issue in taboola.json where the iframePing needs to be setup

### PR DESCRIPTION
**Pull request by udaya-m-taboola**
@micajuine-ho This pull request is to fix the bug in https://github.com/ampproject/amphtml/pull/32803. Found an issue where our trk.js was not getting loaded and after debugging found out it is because of missing "iframePing":true. Can you please take a look at the change to fix that. We need this to be resolved, since we have customers waiting for this fix. Thanks

### CODE REVIEW

**Comment by micajuine-ho** [comment]
Sorry, can you explain the situation a little more? 
As mentioned in `#32803`, `iframeping` is a workaround, and so shouldn't be used generally. I'm very open to allowing it, but I'd like to understand the context a little more. Thanks

File: extensions/amp-analytics/0.1/vendors/taboola.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
   "triggers": {
     "pageview": {
       "on": "visible",
-      "request": "loadamptrkhtml"
+      "request": "loadamptrkhtml",
+      "iframePing":true
```
Comment (micajuine-ho) [inline extensions/amp-analytics/0.1/vendors/taboola.json:12]: nit: s/`"iframePing":true`/`"iframePing": true`

**Review by micajuine-ho — CHANGES_REQUESTED** [review]

**Comment by udaya-m-taboola** [comment]
Opened a new pull request on a new branch https://github.com/ampproject/amphtml/pull/34007

**Pull request closed without merging**